packages/platforms/react-native/ios/BugsnagReactNativePerformance.mm

```diff
@@ -82,12 +82,118 @@ @implementation BugsnagReactNativePerformance
     return hexStr;
 }
 
- RCT_EXPORT_METHOD(requestEntropyAsync:(RCTPromiseResolveBlock)resolve
-                   rejecter:(RCTPromiseRejectBlock)reject)
- {
-     NSString *hexStr = getRandomBytes();
-     resolve(hexStr);
- }
+RCT_EXPORT_METHOD(requestEntropyAsync:(RCTPromiseResolveBlock)resolve
+                   rejecter:(RCTPromiseRejectBlock)reject) {
+    NSString *hexStr = getRandomBytes();
+    resolve(hexStr);
+}
+
+RCT_EXPORT_BLOCKING_SYNCHRONOUS_METHOD(getNativeConstants) {
```

Comment:
It's actually a temporary method - it's supposed to be getConstants which uses react native special method for delivering static constants from native side. 
But for now I am using this method. It's not having resolve/reject because it's synchronous.